app/models/decision.rb

```diff
@@ -33,21 +35,29 @@ class Decision < ApplicationRecord
     rejected: 1
   }
 
+  def self.rejected_reasons_for(policy, prefix: nil)
```

Comment:
`prefix` seems to be presentational so I suggest moving the adding of the prefix back to https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2492/files#diff-97f8a78688ed92b6ad6239a753880d1242ab506e93381578f7342d5c11f39decR92

Reply:
@slorek good point, addressed with [928efc8](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2492/commits/928efc851b86475a126925a70795a1f0720e0783)